tests/testthat/test-maic_anchored.R

```diff
@@ -89,184 +34,23 @@ test_that("maic_anchored works for TTE using robust SE", {
   expect_equal(
     result$inferential$report_median_surv$rmean,
     c(
-      2.56479654878613, 8.70968971110584, 2.6906650526407, 10.5753013034989,
-      2.45527171390661, 4.30355056953339
+      2.5647965487863, 8.7096897111058, 2.6794733007394, 10.5846090795552,
+      2.4552717139068, 4.3035505695334
     )
   )
   expect_equal(
     result$inferential$report_median_surv$`se(rmean)`,
     c(
-      0.113669935856185, 0.35514766015862, 0.207503727490195, 0.573259024729393,
-      0.0984888793057228, 0.336726020204787
+      0.113669935856188, 0.355147660158620, 0.206708266956460, 0.573979370096874,
+      0.098488879305725, 0.336726020204787
     )
   )
   expect_equal(
     result$inferential$report_overall_robustCI$`median[95% CI]`,
-    c("7.6[6.3;10.3]", "1.8[1.6; 2.0]", "12.2[10.2; NA]", " 1.8[ 1.5;2.4]", "2.7[2.3;3.3]", "1.9[1.7;2.0]", "--")
+    c("7.6[6.3;10.3]", "1.8[1.6; 2.0]", "12.2[10.2; NA]", " 1.8[ 1.5;2.3]", "2.7[2.3;3.3]", "1.9[1.7;2.0]", "--")
   )
   expect_equal(
     result$inferential$report_overall_robustCI$`HR[95% CI]`,
-    c("0.22[0.19;0.26]", "", "0.16[0.11;0.24]", "", "0.57[0.48;0.68]", "", "0.29 [0.19; 0.44]")
-  )
-})
-
-
-test_that("maic_anchored works for TTE using bootstrap SE", {
```

Comment:
No test on the bootstrap results anymore?

Reply:
oh I deleted some parts.. I'll put them back in